### PR TITLE
Add new FAQ section "Why does my interface look weird?"

### PR DESCRIPTION
**Pull request by mrdrogdrog**
### Component/Part
Documentation - FAQ

### Description
This PR adds a new FAQ entry.

### Steps

- [x] Added / updated documentation
- [x] I read the [contribution documentation](https://github.com/hedgedoc/hedgedoc/blob/develop/CONTRIBUTING.md) and
  made sure that:
  - My commits are signed-off to accept the DCO.
  - This PR targets the correct branch: `master` for 1.x & docs, `develop` for 2.x

### CODE REVIEW

**Comment by davidmehren** [comment]
I'd prefer to put the longer explanation and examples somewhere else. Maybe the reverse proxy guide is a good place?

**Comment by mrdrogdrog** [comment]
What about the example that doesn't use a reverse proxy?

**Comment by mrdrogdrog** [comment]
What about a new article in the configuration namespace?

**Review by InnayTool — APPROVED** [review]

File: docs/content/faq.md [hunk 1]
```diff
@@ -58,3 +58,37 @@ and make sure that `protocolUseSSL` / `CMD_PROTOCOL_USESSL` is set to `true` if
...
+
+If your interface looks like this
+
+![Broken HedgeDoc interface](./faq/interface_broken.png)
+
+instead of this
+
+![Not broken HedgeDoc interface](./faq/interface_okay.png)
+
+then you should check the following settings (or their corresponding environment variables if you e.g. use docker.):
```
Comment (ErikMichelson) [inline docs/content/faq.md:72]:
```suggestion
then you should check the following settings (or their corresponding environment variables if you e.g. use docker):
```

File: docs/content/faq.md [hunk 2]
```diff
@@ -58,3 +58,37 @@ and make sure that `protocolUseSSL` / `CMD_PROTOCOL_USESSL` is set to `true` if
...
+
+- `domain`
+- `protocolUseSSL`
+- `urlAddPort`
```
Comment (ErikMichelson) [inline docs/content/faq.md:76]: Maybe just add the environment variables here too, as we recommend the docker setup and that uses them.

```suggestion
- `domain` (env: `CMD_DOMAIN`)
- `protocolUseSSL` (env: `CMD_PROTOCOL_USESSL`)
- `urlAddPort` (env: `CMD_URL_ADDPORT`)
```

File: docs/content/faq.md [hunk 3]
```diff
@@ -58,3 +58,37 @@ and make sure that `protocolUseSSL` / `CMD_PROTOCOL_USESSL` is set to `true` if
...
+
+Check [the configuration page](/configuration/) for an explanation.
+
+These settings control how HedgeCoc generates links to other pages and to assets, therefore you must set them from the outside point of view.
+
+Example: You access your HedgeDoc instance directly via `http://93.184.216.34:5812`. You must set:
+
+- `domain` to `93.184.216.34` because that's the host.
```
Comment (ErikMichelson) [inline docs/content/faq.md:84]: Use one of the addresses from the following subnets: 

- 192.0.2.0/24
- 198.51.100.0/24
- 203.0.113.0/24

These networks are reserved for documentation and examples.

File: docs/content/faq.md [hunk 4]
```diff
@@ -58,3 +58,37 @@ and make sure that `protocolUseSSL` / `CMD_PROTOCOL_USESSL` is set to `true` if
...
+
+- `domain`
+- `protocolUseSSL`
+- `urlAddPort`
+
+Check [the configuration page](/configuration/) for an explanation.
+
+These settings control how HedgeCoc generates links to other pages and to assets, therefore you must set them from the outside point of view.
+
+Example: You access your HedgeDoc instance directly via `http://93.184.216.34:5812`. You must set:
```
Comment (ErikMichelson) [inline docs/content/faq.md:82]: We should probably keep `3000` as the example port as this is the default port for HedgeDoc.

File: docs/content/faq.md [hunk 5]
```diff
@@ -58,3 +58,37 @@ and make sure that `protocolUseSSL` / `CMD_PROTOCOL_USESSL` is set to `true` if
...
+
+- `domain` to `93.184.216.34` because that's the host.
+- `protocolUseSSL` to `false` because you access your instance via HTTP instead of HTTPS.
+- `urlAddPort` to `true` because you access the instance using a custom port instead of the default HTTP port 80.
+
+Example: You access your HedgeDoc instance using a reverse proxy via `https://markdown.pro`. You must set:
```
Comment (ErikMichelson) [inline docs/content/faq.md:88]: Use `.example` as tld for the domain. This is explicitly reserved for examples and will never be assigned.

See the [RFC 2606](https://datatracker.ietf.org/doc/html/rfc2606#section-2) for this.

**Review by ErikMichelson — CHANGES_REQUESTED** [review]
Aside my comments, the broken interface screenshot is taken in German and not English.

File: docs/content/faq.md (hunk 3 above)
Comment (mrdrogdrog) [reply docs/content/faq.md:84]: This ip is example.org

File: docs/content/faq.md [hunk 6]
```diff
@@ -58,3 +58,38 @@ and make sure that `protocolUseSSL` / `CMD_PROTOCOL_USESSL` is set to `true` if
...
+then HedgeDoc can't find the asset files like images, stylesheets, fonts, etc.
+You should check the following settings (or their corresponding environment variables if you e.g. use docker.):
+
+- `domain` (env: `CMD_DOMAIN`)
+- `protocolUseSSL` (env: `CMD_PROTOCOL_USESSL`)
+- `urlAddPort` (env: `CMD_URL_ADDPORT`)
+
+Check [the configuration page](/configuration/) for an explanation.
+
+These settings control how HedgeCoc generates links to other pages and to assets, therefore you must set them from the outside point of view.
```
Comment (InnayTool) [inline docs/content/faq.md:81]:
```suggestion
These settings control how HedgeDoc generates links to other pages and to assets, therefore you must set them from the outside point of view.
```
@mrdrogdrog HedgeCoc as a HedgeDoc code of conduct have to be on the list.

**Review by InnayTool — CHANGES_REQUESTED** [review]

**Review by InnayTool — APPROVED** [review]

File: docs/content/faq.md [hunk 7]
```diff
@@ -58,3 +58,38 @@ and make sure that `protocolUseSSL` / `CMD_PROTOCOL_USESSL` is set to `true` if
...
+
+These settings control how HedgeDoc generates links to other pages and to assets, therefore you must set them from the outside point of view.
+
+Example: You access your HedgeDoc instance directly via `http://192.0.2.1:3000`. You must set:
+
+- `domain` to `192.0.2.1` because that's the host.
+- `protocolUseSSL` to `false` because you access your instance via HTTP instead of HTTPS.
+- `urlAddPort` to `true` because you access the instance using a custom port instead of the default HTTP port 80.
+
+Example: You access your HedgeDoc instance using a reverse proxy via `https://markdown.example`. You must set:
```
Comment (DerMolly) [inline docs/content/faq.md:89]:
```suggestion
Example: You access your HedgeDoc instance using a reverse proxy via `https://markdown.example.com`. You must set:
```

File: docs/content/faq.md [hunk 8]
```diff
@@ -58,3 +58,38 @@ and make sure that `protocolUseSSL` / `CMD_PROTOCOL_USESSL` is set to `true` if
...
+
+- `domain` to `markdown.pro`.
```
Comment (DerMolly) [inline docs/content/faq.md:91]:
```suggestion
- `domain` to `markdown.example.com`.
```

File: docs/content/faq.md [hunk 9]
```diff
@@ -58,3 +58,38 @@ and make sure that `protocolUseSSL` / `CMD_PROTOCOL_USESSL` is set to `true` if
...
+
+- `domain` (env: `CMD_DOMAIN`)
+- `protocolUseSSL` (env: `CMD_PROTOCOL_USESSL`)
+- `urlAddPort` (env: `CMD_URL_ADDPORT`)
+
+Check [the configuration page](/configuration/) for an explanation.
+
+These settings control how HedgeDoc generates links to other pages and to assets, therefore you must set them from the outside point of view.
+
+Example: You access your HedgeDoc instance directly via `http://192.0.2.1:3000`. You must set:
```
Comment (DerMolly) [inline docs/content/faq.md:83]: Does our hugo theme support alert blocks like HedgeDoc does? Maybe these examples would be a good canidate for this.

**Review by DerMolly — CHANGES_REQUESTED** [review]

File: docs/content/faq.md (hunk 9 above)
Comment (mrdrogdrog) [reply docs/content/faq.md:83]: It does afaik but I don't know if that's necessary here.

**Review by ErikMichelson — APPROVED** [review]

File: docs/content/faq.md (hunk 9 above)
Comment (mrdrogdrog) [reply docs/content/faq.md]: I added example blocks

**Review by DerMolly — APPROVED** [review]

**Review by DerMolly — APPROVED** [review]

**Review by ErikMichelson — APPROVED** [review]

**Review by InnayTool — APPROVED** [review]

**Review by davidmehren — APPROVED** [review]

**Pull request merged**